codecparsers/bitReader_unittest.cpp

```diff
@@ -48,12 +48,12 @@ BITREADER_TEST(NoOverflow)
     std::vector<uint8_t>& data = largeData();
     const uint32_t nBytes = data.size();
 
-    ASSERT_EQ(0, nBytes % (1024 * 1024 * 512));
+    ASSERT_EQ(0u, nBytes % (1024 * 1024 * 512));
```

Comment:
It's very incontinence to add 'u' prefix for all  unsigned value. And it's hard to maintain.It's very likely we have someone do not use gcc 5.3.1 and did not pay attention to the singed unsigned comparison. He will introduce some unit test which break build for gcc 5.3.1.
Is it ok to ignore signed/unsigned compare warning in unit test?